src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -1074,12 +1069,12 @@ msgid "Notification preferences"
 msgstr "Notificatievoorkeuren"
 
 #: open_inwoner/accounts/templates/accounts/registration_necessary.html:68
-#: open_inwoner/templates/pages/profile/notifications.html:47
+#: open_inwoner/templates/pages/profile/notifications.html:49
 msgid "Zaaknotificaties - actie is nodig"
-msgstr "Zaaknotificaties - actie is nodig"
+msgstr "Zaaknotificaties indien actie nodig is"
```

Comment:
@alextreme We hebben nog geen antwoord op de vraag wat dit moet gaan worden maar vandaag is de laatste dag dat ik aan deze PR kan werken 😏

Reply:
@jiromaykin geef ze tot maandag, en anders houden we het zoals het is voor v1.21

Reply:
Voorlopig: "Zaaknotificaties wanneer actie nodig is"